sso-kit-client/lit/package.json

```diff
@@ -55,7 +55,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@hilla/frontend": "2.0.5",
+    "@vaadin/hilla-core": "24.4.0-alpha1",
```

Comment:
An ongoing PR in Hilla is about renaming `@vaadin/hilla-core` to `@vaadin/hilla-frontend` [#2052](https://github.com/vaadin/hilla/pull/2052), so it might be better to already refactor it to `@vaadin/hilla-frontend` and wait a bit until next alpha released.

Reply:
Using 24.4.0-alpha4 now with `@vaadin/hilla-frontend` 👍